lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1033,6 +1033,7 @@ class MongoClientInterface {
         const masterKey = formatMasterKey(objName, vFormat);
         MongoUtils.serialize(objVal);
         // eslint-disable-next-line
+        objVal.originOp = 's3:ObjectRemoved:Delete';
```

Comment:
It seems the `eslint-disable` isn't necessary any longer since it isn't complaining any longer

Reply:
just tried it's still complaining